app/routes/conf.tsx

```diff
@@ -82,6 +86,20 @@ export default function Component() {
           <div className="flex flex-col space-y-4 p-4 text-gray-200 sm:space-y-8 sm:p-8">
             <Outlet />
           </div>
+          <div className="mt-8 flex flex-row justify-between p-4 pb-8 sm:mt-0 sm:p-8">
+            {previous ? (
+              <SecondaryButtonLink to={$path(previous)}>
```

Comment:
I don't think `$path` does anything here. It only works for absolute paths. Relative paths are not type-safe at the moment. Can you remove it, please?